tests/testthat/test-mask.R

```diff
@@ -0,0 +1,119 @@
+require(Matrix)
```

Comment:
Can we also test whether simil() is equivalent when `mask = NULL` vs. when `mask = mask(c("a", "a", "a", "a", ...), c("a", "a", "a", "a", ...))` ?

Reply:
Thanks. I added tests to compare `mask = NULL` and `mask` is matrix with all `TRUE`.

Reply:
Also added a vignette page. Please feel free to edit.

Reply:
Thanks! :bow: